specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/examples/DatabaseBlobAuditingCreateMax.json

```diff
@@ -0,0 +1,57 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "blobauditingtest-4799",
+    "serverName": "blobauditingtest-6440",
+    "databaseName": "testdb",
+    "blobAuditingPolicyName": "default",
+    "api-version": "2017-03-01-preview",
+    "parameters": {
+      "properties": {
+          "state": "Enabled",
+          "storageAccountAccessKey": "sdlfkjabc+sdlfkjsdlkfsjdfLDKFTERLKFDFKLjsdfksjdflsdkfD2342309432849328476458/3RSD==",
+          "storageEndpoint": "https://mystorage.blob.core.windows.net",
+          "retentionDays": 6,
+          "storageAccountSubscriptionId": "00000000-1234-0000-5678-000000000000",
+          "isStorageSecondaryKeyInUse": false,
+          "auditActionsAndGroups":["DATABASE_LOGOUT_GROUP","DATABASE_ROLE_MEMBER_CHANGE_GROUP", "UPDATE on database::TestDatabaseName by public"]
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/blobauditingtest-4799/providers/Microsoft.Sql/servers/blobauditingtest-6440/databases/testdb",
+        "name": "default",
```

Comment:
>  "name": "default", [](start = 6, length = 20)

Does your service always return `default` value for Name parameter in response to PUT operation?

Reply:
This is a "singleton resource", so "default" is the only valid name.